app/views/markets/index.html.erb

```diff
@@ -0,0 +1,30 @@
+
+
+<h1>Markets</h1>
+
+<div class="market_button">
+	<%= link_to new_market_path, class: 'btn btn-success' do %>
+	  Add a New Market
+	<% end %>
+</div>
+<br>
+<br>
```

Comment:
Adding ```<br>``` is generally not good practice for creating space. A better way to do this is to adjust the padding or margins of the elements around it